cmds/minigames/connect4.js

```diff
@@ -1,84 +1,141 @@
-                    let pass = game.reactions.includes(reaction.emoji.name) && !user.bot && user.id === game.yellowid
-                    return pass
+                // since the bot is sending embed in DM, you don't need to check for user id
+                const filter = (reaction, user) => {
+                    let pass = game.reactions.includes(reaction.emoji.name) && !user.bot;
+                    return pass;
                 }
 
-                const redcollector = rmsg.createReactionCollector(rfilter,{ idle: 300*1000 })
-                const yellowcollector = ymsg.createReactionCollector(yfilter,{ idle: 300*1000 })
-
-                redcollector.on("collect", async (reaction,user) =>{
-                
-                    if(reaction.emoji.name === "🛑"){
-
-                        rmsg.edit(client.EmbedMaker(msg, client.language.commands.game[userlang].abrupted.replace("{0}",user.username), client.colors.black))
-                        ymsg.edit(client.EmbedMaker(msg, client.language.commands.game[userlang].abrupted.replace("{0}",user.username), client.colors.black))
-                        end = true
-                        redcollector.stop()
-                        yellowcollector.stop()
-                        return
-
+                const redcollector = rmsg.createReactionCollector(filter, {
+                    idle: 300 * 1000,
+                });
+                const yellowcollector = ymsg.createReactionCollector(filter, {
+                    idle: 300 * 1000,
+                });
+
+                // merge 2 collector if possible
+                redcollector.on("collect", async (reaction, user) => {
+                    if (reaction.emoji.name === "🛑") {
+                        rmsg.edit(
+                            client.EmbedMaker(
+                                msg,
+                                client.language.commands.game[userlang].abrupted.replace("{0}",user.username),
+                                client.colors.black
+                            )
+                        );
+                        ymsg.edit(
+                            client.EmbedMaker(
+                                msg,
+                                client.language.commands.game[userlang].abrupted.replace("{0}",user.username),
+                                client.colors.black
+                            )
+                        );
+                        end = true;
+                        redcollector.stop();
+                        yellowcollector.stop();
+                        return;
                     }
 
-                    console.log("r1")
-                    if(user.id === game.redid && current_color !== "red") return
-                    console.log("r2")
-
-                    switch(game.place("1️⃣2️⃣3️⃣4️⃣5️⃣6️⃣7️⃣🛑".indexOf(reaction.emoji.name)/3,current_color)){
-
+                    console.log("r1");
+                    if (user.id === game.redid && current_color !== "red")
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/xiaomiao-fong/Chen_bot/pullRequest?prid=7807535)
```suggestion
                    if (user.id === game.redid && current_color !== "red")
```